crypto/ocsp/ocsp_test.cc

```diff
@@ -1380,3 +1380,66 @@ TEST(OCSPTest, CertIDDup) {
   // Check that the duplicated structure is not just a replicated pointer.
   EXPECT_NE(orig_id.get(), dup_id.get());
 }
+
+TEST(OCSPTest, OCSPResponsePrint) {
+  static const std::string kExpected[] = {
+      "OCSP Response Data:",    "    OCSP Response Status:",
+      "    Response Type:",     "    Version:",
+      "    Responder Id:",      "    Produced At:",
+      "    Responses:",         "    Certificate ID:",
+      "      Hash Algorithm:",  "      Issuer Name Hash:",
+      "      Issuer Key Hash:", "      Serial Number:",
+      "    Cert Status:",       "    This Update:",
+      "    Next Update:",       "    Response Extensions:"};
+
+  std::string respData = GetTestData(
+      std::string("crypto/ocsp/test/aws/ocsp_response.der").c_str());
+  std::vector<uint8_t> ocsp_response_data(respData.begin(), respData.end());
+  bssl::UniquePtr<OCSP_RESPONSE> ocspResponse =
+      LoadOCSP_RESPONSE(ocsp_response_data);
+
+  bssl::UniquePtr<BIO> bio(BIO_new(BIO_s_mem()));
+  EXPECT_TRUE(OCSP_RESPONSE_print(bio.get(), ocspResponse.get(), 0));
+  char *out;
+  ASSERT_TRUE(BIO_get_mem_data(bio.get(), &out));
+
+  // Iterate through |kExpected| and verify that |OCSP_RESPONSE_print| has
+  // the expected format.
+  char *token = strtok(out, "\n");
```

Comment:
Since this is C++ you could avoid strtok and use some safer options like [std::istringstream](https://en.cppreference.com/w/cpp/io/basic_istringstream):
```
 std::istringstream out_stream(string(out));
 std::string line;
 while (std::getline(out_stream, line)) {
    // Check they're equal
}
```

Reply:
Changed to use std::istringstream instead, thanks for the tip :)